src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/ConvertToJsonCommand.cs

```diff
@@ -100,6 +100,12 @@ protected override void BeginProcessing()
         /// </summary>
         protected override void ProcessRecord()
         {
+            if (InternalTestHooks.ActivateSleepForStoppingTest)
+            {
+                WriteVerbose("ConvertTo-Json started");
+                System.Threading.Thread.Sleep(50);
```

Comment:
I don't think this sleep is needed.  There was a concern that the cmdlet would finish successfully before the stop of the pipeline got issued.  I think they way to solve that is to have another check at the end of ProcessRecord() where if the test hook is enabled, just have it sleep and not finish.

Reply:
Stopping doesn't work on sleeping runspace. So idea is only slow down the cycle and make many interations.

Reply:
Ok, perhaps add a comment?

Reply:
Done.